src/pages/create-session/CreateSession.tsx

```diff
@@ -27,12 +29,17 @@ const steps = [
 
 const CreateSession = () => {
   const history = useHistory();
-  const [session, setSession] = useState<{
-    name: string;
-    startDate: Date | null;
-  }>({
+  const [session, setSession] = useState<SessionRequest>({
     name: "",
     startDate: null,
+    classes: [
+      {
+        name: "",
+        days: [],
+        location: "",
+        facilitator: null,
+      },
```

Comment:
nit: can we reference `defaultClassData` in the ClassForm component for this?

Reply:
yeah, missed that. thanks!